debugging.rst

```diff
@@ -0,0 +1,84 @@
+.. _debugging:
+
+==============
+Debugging Jobs 
+==============
+
+Flux supports parallel debuggers such as Rogue Wave Software (RWS)'s
+TotalView parallel debugger (https://www.roguewave.com).
```

Comment:
ReST has support for external links.  The syntax is weird:

```suggestion
Flux supports parallel debuggers such as `Rogue Wave Software (RWS)'s <https://www.roguewave.com>`_
TotalView parallel debugger.
```

Reply:
Learned a new thing! Will change.